style/checkstyle.xml

```diff
@@ -275,13 +275,13 @@
         <module name="EmptyStatement" />
 
         <!-- Checks for Java Docs. -->
-        <module name="JavadocStyle"/>
+<!--        <module name="JavadocStyle"/>-->
```

Comment:
Make sure whether it's your local change?

Reply:
sorry, I didn't notice this change.

Reply:
So let's revert?

Reply:
right, done.